examples/FluentUI.Demo.Shared/Pages/CardPage.razor

```diff
@@ -1,114 +1,113 @@
 @page "/CardPage"
-<FluentDesignSystemProvider use-defaults background-color="#F7F7F7" corner-radius="10">
-    <style>
-        fluent-card {
-            --card-height: 400px;
-            --card-width: 500px;
-            padding: 20px;
-            margin: 12px;
-        }
 
-        .class-override {
-            height: 163px;
-            width: 300px;
-        }
+<style>
+    fluent-card {
+        --card-height: 400px;
+        --card-width: 500px;
+        padding: 20px;
+        margin: 12px;
+    }
 
-        .state-override {
-            --card-width: 350px;
-            --card-height: 300px;
-            --elevation: 6;
-        }
+    .class-override {
+        height: 163px;
+        width: 300px;
+    }
 
-            .state-override:hover {
-                --elevation: 12;
-            }
+    .state-override {
+        --card-width: 350px;
+        --card-height: 300px;
+        --elevation: 6;
```

Comment:
Setting `--elevation` used to be the way the card got its shadow, but this has been replaced with direct tokens like `elevationShadowCardRest`.

Reply:
Where is `elevationShadowCardRest` defined? I've only looked at DesignTokens.ts for which tokens to generate.